components/mediation-initializer/org.wso2.carbon.mediation.initializer/src/main/java/org/wso2/carbon/mediation/initializer/ServiceBusConstants.java

```diff
@@ -71,6 +71,7 @@
 
     public static final String ARTIFACT_EXTENSION = "xml";
     public static final String CLASS_MEDIATOR_EXTENSION = "jar";
+    public static final String SYNAPSE_LIBRARY_EXTENSION = "zip";
```

Comment:
Artifact and class mediator extensions are added here. Likewise I added the extension for the synapse library here.